src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,20 @@
+@import "./variables";
+
+@mixin element-size ($height , $width) {
+  height: $height;
+  width: $width;
+}
+
+@mixin position ($left , $top) {
+  left: $left;
+  top: $top;
+}
```

Comment:
Overuse of mixin is not a good practice, in my opinion your code would be more readable if you could simply add those properties directly to elements